tools/travis/activat.ion

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?><root>
+    <License id="Terms">
+        <MachineBindings>
+            <Binding Key="1" Value=""/>
+        </MachineBindings>
+        <MachineID Value="2jmj7l5rSw0yVb/vlWAYkK/YBwk="/>
+        <SerialHash Value="734ab3895f13da29342af6444199b076ee4741eb"/>
+        <Features>
+            <Feature Value="33"/>
+            <Feature Value="1"/>
+            <Feature Value="12"/>
+            <Feature Value="2"/>
+            <Feature Value="24"/>
+            <Feature Value="3"/>
+            <Feature Value="36"/>
+            <Feature Value="17"/>
+            <Feature Value="19"/>
+            <Feature Value="62"/>
+        </Features>
+        <DeveloperData Value="AQAAAEY0LVI4VEstNU1SOS1UVUNVLUtXOFgtWUM1Mg=="/>
+        <SerialMasked Value="F4-R8TK-5MR9-TUCU-KW8X-XXXX"/>
+        <StartDate Value="2017-12-16T14:55:19"/>
+        <UpdateDate Value="2018-11-02T23:05:29"/>
+        <InitialActivationDate Value="2017-12-16T14:55:19"/>
+        <LicenseVersion Value="6.x"/>
+        <ClientProvidedVersion Value="2018.2.6f1"/>
```

Comment:
See the other comment about the Unity version.

Reply:
Wasn't able to get the older versions to work with travis-ci. It supports the desired testing + compile checks for CI purposes of the 2018.1.4f1 version project files. Also, see #28

Reply:
Green light for using it then. I use 2018.2.0f2 myself, and as long as I don't commit anything involving .meta files, I haven't noticed any issues on the current .NET.